gold-phone-input.html

```diff
@@ -157,9 +157,27 @@
       }
     },
 
-    observers: [
-      '_computeValue(value)'
-    ],
+    listeners: {
+      'input': '_onInput'
+    },
+
+    ready: function() {
+      this.addEventListener('blur', function() {
+        this._handleAutoValidate();
+      }, true);
+
+      // If there's an initial input, validate it.
+      if (this.value)
+        this._handleAutoValidate();
+    },
+
+    /**
+     * If `autoValidate` is true, then validates the element.
+     */
+    _handleAutoValidate: function() {
```

Comment:
Sooooooo yes, it does, but ideally I'd like this to be an argument about gold elements being an "is-a" paper-input (hence my talks about trying to reuse the `paper-input` template). If you look at `paper-input-container`, it knows how to use `auto-validate` just fine, but not when it doesn't get to control the validation like this.